package.json

```diff
@@ -118,6 +118,32 @@
         "mac": "shift+f5",
         "when": "editorLangId == python"
       }
+    ],
+    "taskDefinitions": [
+			{
+				"type": "austin",
+				"required": [
+					"file"
+				],
+				"properties": {
+					"file": {
+						"type": "string",
+						"description": "The python file path to profile."
+					},
+					"args": {
+						"type": "array",
+						"description": "Optional list of arguments to the task"
+					},
+          "austinArgs": {
+						"type": "array",
+						"description": "Optional list of austin command-line options"
+					},
+          "profileName": {
+            "type": "string",
+            "description": "Name of the generated profile"
+          }
+				}
```

Comment:
I wonder if there should perhaps be another task definition that would handle cases like `pipenv` or `poetry`, e.g.

~~~
poetry run austin ...
~~~

Reply:
It would run from the virtualenv anyway, but there needs to be a case for running a module instead of just a file